internal/cmd/oohelperd/internal/websteps/factory.go

```diff
@@ -1,96 +0,0 @@
-package websteps
```

Comment:
For the records: this file has been deleted because a similar functionality was in the websteps experiment